page/view-news.php

```diff
@@ -0,0 +1,106 @@
+<link rel="stylesheet" href="css/template-1.css">
+<div class="full text-center">
+    <span class="garis"></span>
+    <h1 class="display-5 text-light"><dt><b>BERITA</b></dt></h1>
+</div>
+
+<div class="container mt-5">
+    <div class="row">
+        <div class="col-md-8">
+            <a href="#" id="back-btn" class="btn btn-warning text-light mb-5" onclick="goBack()"><i class="fa fa-chevron-left" aria-hidden="true"></i> BACK</a>
+
+            <?php
+            
+            $select = mysqli_query($connection, "select * from berita where id_berita=".$_GET["id"]);
```

Comment:
Codacy found an issue: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)